# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/Send-binary-arraybufferview-uint16-offset-length.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/websockets/Send-binary-arraybufferview-uint16-offset-length.any.js.ini
--- a/testing/web-platform/meta/websockets/Send-binary-arraybufferview-uint16-offset-length.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/websockets/Send-binary-arraybufferview-uint16-offset-length.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,32 +1,29 @@
 [Send-binary-arraybufferview-uint16-offset-length.any.worker.html?wpt_flags=h2]
   expected:
-    if (os == "linux") and webrender and debug and not swgl: TIMEOUT
-    if (os == "linux") and webrender and debug and swgl: ["TIMEOUT", "OK"]
-    if (os == "linux") and not webrender and not debug: ["OK", "TIMEOUT"]
-    if (os == "linux") and not webrender and debug: ["OK", "TIMEOUT"]
-    if (os == "linux") and webrender and not debug: ["TIMEOUT", "ERROR"]
+    if (os == "linux") and webrender and debug and swgl: ["OK", "TIMEOUT"]
+    if (os == "linux") and webrender and not debug: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "linux") and not webrender: ["OK", "TIMEOUT"]
   [Send binary data on a WebSocket - ArrayBufferView - Uint16Array with offset and length - Connection should be opened]
     expected:
+      if (os == "linux") and debug and webrender: ["PASS", "NOTRUN"]
       if (os == "linux") and debug and not webrender: ["PASS", "NOTRUN"]
-      if (os == "linux") and debug and webrender: ["NOTRUN", "PASS"]
 
   [Send binary data on a WebSocket - ArrayBufferView - Uint16Array with offset and length - Message should be received]
     expected:
-      if (os == "linux") and debug and webrender: ["NOTRUN", "PASS"]
       if (os == "linux") and debug and not webrender: ["PASS", "NOTRUN"]
+      if (os == "linux") and debug and webrender: ["PASS", "NOTRUN"]
 
   [Send binary data on a WebSocket - ArrayBufferView - Uint16Array with offset and length - Connection should be closed]
     expected:
       if (os == "linux") and debug and not webrender: ["PASS", "FAIL"]
-      if (os == "linux") and debug and webrender: ["FAIL", "PASS"]
+      if (os == "linux") and debug and webrender: ["PASS", "FAIL"]
 
 
 [Send-binary-arraybufferview-uint16-offset-length.any.html?wpt_flags=h2]
   expected:
-    if (os == "linux") and webrender and debug and swgl: ["OK", "TIMEOUT"]
-    if (os == "linux") and webrender and debug and not swgl: TIMEOUT
-    if (os == "linux") and not webrender and debug: ["TIMEOUT", "OK"]
-    if (os == "linux") and webrender and not debug: ["TIMEOUT", "ERROR"]
+    if (os == "linux") and not swgl and not debug and webrender: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "linux") and not swgl and debug and not webrender: ["OK", "TIMEOUT"]
+    if (os == "linux") and swgl: ["OK", "TIMEOUT"]
 
 [Send-binary-arraybufferview-uint16-offset-length.any.worker.html?wss]
 